README.rst

```diff
@@ -120,6 +123,47 @@ Options
                             containing the private key and the certificate in PEM
                             format
 
+    $ twine register -h
```

Comment:
Register needs to be deprecated and is presently broken (as pypi.org disabled that API). I'm wondering if we should document this anyway or if we should omit it.

Reply:
My inclination is to document it (and to mention that it is broken and about to be deprecated) until we actually deprecate it, and *then* to remove the documentation.